Marlin/src/module/planner.h

```diff
@@ -267,6 +267,7 @@ typedef struct PlannerBlock {
              final_adv_steps;               // Advance steps for exit speed pressure
   #endif
 
+  #define MINIMAL_STEP_RATE _MAX((STEPPER_TIMER_RATE / HAL_TIMER_TYPE_MAX), 1)   // steps/s max. To prevent timer overflow, slowest is 1 step/s
```

Comment:
I think it is not necessary to do this. `Stepper::calc_timer_interval()` has it covered.